extensions/apps/antenna/src/components/world-version-info-card/index.tsx

```diff
@@ -2,33 +2,31 @@ import React from 'react';
 import Button from '@akashaorg/design-system-core/lib/components/Button';
 import { TriangleAlertIcon, XIcon } from 'lucide-react';
 import { Stack } from '@akashaorg/ui/lib/akasha-components/stack';
-import Text from '@akashaorg/design-system-core/lib/components/Text';
-
+import { Typography } from '@akashaorg/ui/lib/akasha-components/typography';
 export type TWorldVersionInfoCard = {
   titleLabel: string;
   description: string;
   onDismissCard: () => void;
 };
-
 const WorldVersionInfoCard: React.FC<TWorldVersionInfoCard> = props => {
   const { titleLabel, description, onDismissCard } = props;
-
   return (
     <Stack spacing={1} className="p-4 bg-warningLight/30 dark:bg-warningDark/30 rounded-3xl">
       <Stack direction="row" alignItems="center" justifyContent="between">
         <Stack direction="row" alignItems="center" spacing={1}>
+          <Typography variant="sm" bold>
+            {titleLabel}
+          </Typography>
           <TriangleAlertIcon className="h-4 w-4 [&>*]:stroke-warningLight dark:[&>*]:stroke-warningDark" />
```

Comment:
This text now comes before the icon. is this intended

Reply:
Likely an issue introduced during merge conflict resolution, I had to adress a lot of merge conflict

Reply:
Likely an issue introduced during merge conflict resolution, I had to address a lot of them. I have done another commit to address such cases. Please let me know if encounter another one.